cg/meta/report/field_validators.py

```diff
@@ -1,11 +1,19 @@
 """Report field validation helper"""
 
+from typing import Optional
+
 from cg.constants import NA_FIELD
 from cg.models.report.report import ReportModel
 
 
+def get_million_read_pairs(reads: int) -> Optional[float]:
+    """Represents the number of sequencing reads as millions of read pairs."""
+
+    return round(reads / 2_000_000, 1) if reads or isinstance(reads, int) else None
```

Comment:
```suggestion
    return round(reads / SCALE_TO_MILLION_READ_PAIRS, 1) if reads or isinstance(reads, int) else None
```
How about this? At least now one knows what it is supposed to do

Reply:
Included here: https://github.com/Clinical-Genomics/cg/pull/1567